drivers/eaton-pdu-revelation-mib.c

```diff
@@ -173,4 +173,4 @@ static snmp_info_t eaton_aphel_revelation_mib[] = {
 };
 
 
-mib2nut_info_t	aphel_revelation = { "aphel_revelation", EATON_APHEL_REVELATION_MIB_VERSION, NULL, APHEL2_OID_MODEL_NAME, eaton_aphel_revelation_mib, APHEL2_SYSOID };
+mib2nut_info_t	aphel_revelation = { "aphel_revelation", EATON_APHEL_REVELATION_MIB_VERSION, NULL, APHEL2_OID_MODEL_NAME, eaton_aphel_revelation_mib, APHEL2_SYSOID, NULL };
```

Comment:
Given PR #841 and discussion there, I am not sure if other `*-mib.c` files all around NUT codebase have to be amended similarly (or if the mentioned bug in python script should be prioritized)? @aquette - any ideas?